custom_components/powercalc/data/signify/LCX001/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "hass",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.11.7:docker"
+    },
+    "name": "LCX001",
```

This must be the marketed name. i.e.:

```suggestion
    "name": "Hue Gradient Lightstrip",
```